sonar-update-center-mojo/src/test/resources/org/sonar/updatecenter/mojo/GenerateMojoTest/update-center-template/cli.properties

```diff
@@ -0,0 +1,24 @@
+category=Scanners
```

Comment:
You're adding this file, but you haven't updated `GenerateMojoTest.java` to see the result.